int32 type
float64[] info
geometry_msgs/Pose pose
uint16 id

